.github/actions/cl2-modules/netpol/config.yaml

```diff
@@ -58,3 +56,14 @@ steps:
     path: ./modules/metrics.yaml
     params:
       action: gather
+
+- module:
```

Comment:
Nit. In the servicechurn test, objects are deleted before stopping the pprofs/metrics collections, while here that's done after. I wonder if it could make sense aligning the two approaches.

Reply:
So for network policy test I didn't want change results for now - the reason is we have quite a lot of historic data for it. 
At least not for now during release time, but later on, yes - I think it makes sense to switch it.